database/migrations/2018_08_08_100000_create_telescope_entries_table.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
This was missing in the repo, so my 'fresh' database for testing didn't include telescope. This migration has already been run on staging and live, so including it here will not affect the published databases.